DigitalLearningSolutions.Web/Helpers/ViewDataHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    public static class ViewDataHelper
+    {
+        public const string Title = "Title";
+        public const string Application = "Application";
+        public const string DlsSubApplication = "DlsSubApplication";
+        public const string SelectedTab = "SelectedTab";
+        public const string HeaderPath = "HeaderPath";
+        public const string HeaderPathName = "HeaderPathName";
+        public const string HeaderPrefix = "HeaderPrefix";
+        public const string DoNotDisplayNavBar = "DoNotDisplayNavBar";
+        public const string CustomisationId = "CustomisationId";

```

Comment:
Can we call this `CustomisationIdForHeaderLogo` or something similar? That's from a glance at where it's used, but if you could look as well to make sure that makes sense that'd be great.